### PR TITLE
Individual Event Page

### PR DESCRIPTION
**Pull request by JoeStanton**
### Motivation

Port individual event pages from `website-next` to `website-honestly`.

### Test plan

**TODO**

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by asavin** [comment]
LGTM

File: site/routes/definitions.js [hunk 1]
```diff
@@ -40,6 +40,13 @@ export const routeDefinitions : Array<RouteDefinition> = [
     stateToProps: ({ events }) => ({ events }),
   },
   {
+    title: event => event.title,
+    key: 'event',
+    route: 'about-us/events/{day}/{month}/{year}/{slug}',
```
Comment (lazydevorg) [inline site/routes/definitions.js]: Would it be better have {year}/{month}/{day} ?
This way it makes more sense from a URL perspective. The year contains the months and then the days.
It's what usually see on blogs.

Comment (lazydevorg) [inline site/routes/definitions.js]: 👍

File: site/routes/definitions.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const routeDefinitions : Array<RouteDefinition> = [
   {
     title: event => event.title,
     key: 'event',
-    route: 'about-us/events/{day}/{month}/{year}/{slug}',
+    route: 'about-us/events/{year}/{month}/{date}/{slug}',
```
Comment (lazydevorg) [inline site/routes/definitions.js]: typo: `date` => `day`

**Comment by asavin** [comment]
Page titles are always `undefined`

**Pull request merged**